.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -35,6 +35,8 @@ env:
   SCCACHE_BUCKET: ossci-compiler-cache-circleci-v2
   TORCH_CUDA_ARCH_LIST: 5.2
   IN_CI: 1
+  # This is used for the phase of adding wheel tests only, will be removed once completed
```

Comment:
not sure what does it mean when "removed once completed", do you mean unset the `IN_WHEEL_TEST` once the workflow/job it's done, or this env will be removed in the following PRs?

Reply:
also, it sets `IN_WHEEL_TEST: 1` for all the jobs, is there any jobs that don't need to be in `IN_WHEEL_TEST`?

Reply:
The goal is to update all relevant tests to call the wheel install folder instead of the build folder (likely will remove `build/` folder out of `artifacts.zip`) and will change each tests set one by one. `IN_WHEEL_TEST` is used in each tests set to control the change so will need for most of the jobs (currently focusing on linux ci). Once we change all the tests to call the wheel install folder, the flag `IN_WHEEL_TEST` (and relevant checking) could be cleaned up

Reply:
have the same question on when do we want to enalbe `IN_WHEEL_TEST`? the original pitch is to test shipped package and make sure it is smooth. Is there any limitation/con for building wheel on all CI job, if so what will those be and which config should we enabled it; if not, why hasn't it be done previously?

Reply:
My understanding is many of our current test sets are calling/depending on shared library in build/ folder. Torch wheel package is generated and added to artifacts.zip but not the direct object to be tested on all CI jobs. The test script is quite complex so want to make the change gradually, not at once shot.

In this PR and next several PR, I am focusing on Linux CI on GHA only, not circleCI or other OS (windows, mac), so intentionally setting `IN_WHEEL_TEST` for all Linux CI to reduce any risk of break other CI. This work would be expanded to the rest of workflows in the future.